library/xwiki

```diff
@@ -6,13 +6,13 @@ GitCommit: fbc0fe3cbc07119c45ae8ea97c7e021d41e89b0a
 Directory: 8/mysql-tomcat
 
 Tags: 9, 9.7, 9-mysql-tomcat, stable-mysql-tomcat, stable-mysql, stable
```

Comment:
Shouldn't the tags here and below be bumped to `9.8` as well? :wink:

Reply:
grrr.... you're right of course... fixing